### PR TITLE
Added MonoAlphabetic Cipher and updated README.md

### LINKED ISSUE #3803: MonoAlphebatic Cipher in c language
## 🚀 Feature

(A clear and concise description of what the feature is.)
MonoAlphebatic cipher in c language
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?
yes
(Write your answer here.)
 
please assign this to me under GSSoC'21

### PR DESCRIPTION
**Pull request by nagasrihith608**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
yes
### Description

(Write your answer here.)
Added MonoAlphabetic Cipher in C and updated README.md
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixes #3803 
(Write your answer here.)

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format the Code

**Comment by nagasrihith608** [comment]
@HarshCasper , yea done formating please check now

**Comment by nagasrihith608** [comment]
@HarshCasper, I formated the file now please check it.

**Review by HarshCasper — APPROVED** [review]

File: C/cryptography/MonoAlphabeticCipher.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/* Monoalphabetic substitution cipher basically replaces each letter of a plaintext by another letter or symbol, depending only on the letter.
+   The same plaintext letter is always replaced by the same ciphertext letter and the same letter can be replaced by letter itself.
+   Example if a is there we replace it with B to Z and so on for the others.*/
+
+#include<stdio.h>
+#include<string.h>
+
+void monoalphebaticcipher(char input[100],int size){
+     
+     //this where we store output
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     
+     //this where we store output 
+     char output[size-1]; 
+     //this is the list which we use to cipher 
+     char cipherer[26] = {'Z','Y','X','W','V','U','T','S','R','Q','P','O','N','M','L','K','J','I','H','G','F','E','D','C','B','A'};     
+     //the list we use to check which alphebatic we recieved in plain text
+     char list[26] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o','p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'};  
+     //we use this char to compare the space charecter
+     char space = ' ';
+     //this is where we iterate and check what alphabetic is in input and then replace ii with its corresponding cipher
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#include<stdio.h>
+#include<string.h>
+
+void monoalphebaticcipher(char input[100],int size){
+     
+     //this where we store output 
+     char output[size-1]; 
+     //this is the list which we use to cipher 
+     char cipherer[26] = {'Z','Y','X','W','V','U','T','S','R','Q','P','O','N','M','L','K','J','I','H','G','F','E','D','C','B','A'};     
+     //the list we use to check which alphebatic we recieved in plain text
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+   Example if a is there we replace it with B to Z and so on for the others.*/
+
+#include<stdio.h>
+#include<string.h>
+
+void monoalphebaticcipher(char input[100],int size){
+     
+     //this where we store output 
+     char output[size-1]; 
+     //this is the list which we use to cipher
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+     for(int i=0;i<size-1;i++){     
+         for(int j=0;j<26;j++){
+             if(input[i]==list[j])
+                output[i] = cipherer[j];
+             else if( input[i] == space)
+                output[i] = space;    
+         } 
+     } 
+  
+     //printing out the cipher text
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+     printf("The Cipher text :");
+     for(int j=0;j<size-1;j++)
+         printf("%c",output[j]);  
+}
+
+int main(void){
+
+     //we store input here
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

File: C/cryptography/MonoAlphabeticCipher.c [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+     char input[100];
+     printf("Give the plain text here in small letters :");
+     //we take input here
```
Comment (HarshCasper) [inline C/cryptography/MonoAlphabeticCipher.c]: Redundant Comment

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**